packages/capsule/package.json

```
@@ -0,0 +1,67 @@
+{
+  "name": "@web3-onboard/capsule",
+  "version": "1.0.0-alpha.1",
```

```suggestion
  "version": "2.0.0-alpha.1",
```
We are starting all packages on 2 for consistency and alignment with the core modules of W3O versioning 😃